awscli/examples/ecr/get-authorization-token.rst

```diff
@@ -1,58 +1,49 @@
-**To get an authorization token for your default registry**
+**Example 1: To get an authorization token for your default registry**
 
-This example command gets an authorization token for your default registry.
+The following ``get-authorization-token`` example command gets an authorization token for your default registry. ::
 
-Command::
-
-  aws ecr get-authorization-token
+    aws ecr get-authorization-token
 
 Output::
 
-  {
-      "authorizationData": [
-          {
-              "authorizationToken": "QVdTOkN...",
-              "expiresAt": 1448875853.241,
-              "proxyEndpoint": "https://<aws_account_id>.dkr.ecr.us-west-2.amazonaws.com"
-          }
-      ]
-  }
-
+    {
+        "authorizationData": [
+            {
+                "authorizationToken": "QVdTOkN...",
+                "expiresAt": 1448875853.241,
+                "proxyEndpoint": "https://123456789012.dkr.ecr.us-west-2.amazonaws.com"
+            }
+        ]
+    }
 
-**To get the decoded password for your default registry**
+**Example 2: To get the decoded password for your default registry**
 
-This example command gets an authorization token for your default registry and
-returns the decoded password for you to use in a ``docker login`` command.
+The following ``get-authorization-token`` example command gets an authorization token for your default registry and returns the decoded password for you to use in a ``docker login`` command.
```

Comment:
So just today we added a new `get-login-password` command that essentially performs the logic for this example: https://github.com/aws/aws-cli/pull/4874. I'm wondering if we should actually just remove example 2 and example 3 here and point people to use the new command instead? Using the `get-login-password` command is going to be the recommended pattern going forward.